### ISSUE TITLE
cannot install checiopycharmplugin

### ISSUE THREAD

**Issue by smallashash** [evgraph/CheckIOPyCharmPlugin#75]
install the plugin from disk and select the checkiopycharmplugin.zip
restart ..
the plugin still not installed ,i have install jre8 before, where is the logfile

**Comment by evgraph** [comment]
Hi!
Please make sure you're trying to install the latest version of CheckiO plugin (1.0.12) downloaded from JetBrains [repo](https://plugins.jetbrains.com/plugin/7991) and using the latest version of [PyCharm Edu](https://www.jetbrains.com/pycharm-edu/download/) (3.0.1) .

If you still have an installation problems, please attach idea.log file. You could locate it from PyCharm by going to Help-> Show Log In... ([here's](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files) more versbose instruction)

**Comment by smallashash** [comment]
yes, both the plugin and pycharm edu is latest version downloaded from JetBrains , sorry , due to the network connection i can't upload the logfile ,here are some err info in it ,thanks for your help:

Djava.net.preferIPv4Stack=true -XX:+HeapDumpOnOutOfMemoryError -XX:OmitStackTraceInFastThrow 
Cannot run program "git.cmd": CreateProcess error=2, system cannot find the specified file .....
java.io.IOException: Cannot run program "git.cmd": CreateProcess error=2,  system cannot find the specified file.....
Caused by: java.io.IOException: CreateProcess error=2, system cannot find the specified file

**Comment by evgraph** [comment]
@marlow To help you I need a whole log file, if you have problems with attaching it, can you please share all its content via [Pastebin](http://pastebin.com/) for example?

**Comment by smallashash** [comment]
it seems able to work although display with an 'uninstall' , when i click 'create project' i got a warning 'no internet connection'  it that means it can work after solving network problems

**Comment by evgraph** [comment]
Yes, It should work.
